src/aux-server/aux-web/shared/scene/decorators/BotShapeDecorator.ts

```diff
@@ -778,6 +827,18 @@ export class BotShapeDecorator
             this._createCircle();
         } else if (this._shape === 'keyboard') {
             this._createKeyboard();
+        } else if (this._shape === 'light') {
+            if (this._subShape === 'pointLight') {
+                this._createPointLight();
+            } else if (this._subShape === 'ambientLight') {
+                this._createAmbientLight();
+            } else if (this._subShape === 'directionalLight') {
+                this._createDirectionalLight();
+            } else if (this._subShape === 'spotLight') {
+                this._createSpotLight();
+            } else if (this._subShape === 'hemisphereLight') {
+                this._createHemisphereLight();
+            }
```

Comment:
@TroyceGowdy Please add an `else` condition that creates a point light if none of the other conditions match.